documenation/documentation.css

```diff
@@ -3,20 +3,44 @@
 .doc-container {
   display: flex;
   justify-content: center;
-  margin: 6rem;
+  margin: 6rem 6rem 0 6rem;
   position: relative;
 }
 
 .sidebar {
   position: sticky;
   top: 6rem;
+  bottom: 0;
+  left: 0;
+  right: 0;
   display: flex;
   flex: 0.2;
   justify-content: flex-start;
   flex-direction: column;
```

Comment:
you should have used default classes which have already made for `display:flex` `justify-content: flex-start`
  `flex-direction: column`